Apps/Prowlarr/docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
       PGID: "1000"
       PUID: "1000"
       TZ: Europe/London
-    image: linuxserver/prowlarr:0.4.5-nightly
+    image: lscr.io/linuxserver/prowlarr:latest
```

Comment:
Please use specific tag instead of `latest`. Using `latest` adds version uncertainty.